### PR TITLE
Correct specification of NCEP grids 172 and 220

### LINKED ISSUE #157: NCEP grids 172 and 220 not defined correctly in w3fi71.f
A user - @malloryprow - attempted to interpolate a 0.25-degree GFS Grib1 file to NCEP grids 172 and 220 using `copygb` as follows:

```
copygb -g172 -x $gfs_input.grb 172.output.grb
copygb -g220 -x $gfs_input.grb 220.output.grb
```

The output was not on the expected grid because the grid specifications are not being defined properly in `w3fi71.f`. A look at the prolog comments for this routine indicate the definition of the orientation angle for these grids has changed multiple times over the years.

More details of the problem and a solution will be provided.

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
Update w3fi71.f, which specifies the GRIB1 grid description section for NCEP
pre-defined grids, to correct errors with grids 172 and 220.

New unit test for w3fi71.f.

Fixes #157.

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
I added a unit test. But a more robust check would include a mapping of data to grids 172 and 220 using copygb. That could be done here: https://github.com/NOAA-EMC/NCEPLIBS-grib_util/blob/develop/tests/run_copygb_tests.sh

Thoughts? The change to w3emc would need to be coordinated with the change to grib_util.

File: tests/test_w3fi71.f90 [hunk 1]
```diff
@@ -0,0 +1,59 @@
+ program test_w3fi71
```
Comment (edwardhartnett) [inline tests/test_w3fi71.f90:1]: Awesome, thanks for the test!!

**Review by edwardhartnett — APPROVED** [review]

**Comment by edwardhartnett** [comment]
Yes by all means add the test in grib_utils as well.

We can coordinate releases. We have just updated a lot of build stuff having to do with shared vs. static, and we can do a bunch of releases to get the new build stuff out everywhere.

**Comment by edwardhartnett** [comment]
Are you ready for this to me merged? If so, click "Ready for review" and I will merge it.

**Comment by GeorgeGayno-NOAA** [comment]
A test version of copygb was compiled on Cactus using my bug fix branch:
`/lfs/h2/emc/global/noscrub/George.Gayno/mallory/copygb/NCEPLIBS-grib_util/exec/copygb.4.mallory`

@malloryprow just tested it and said it is working.

**Comment by GeorgeGayno-NOAA** [comment]
> Yes by all means add the test in grib_utils as well.

I added a test to copygb. For details, see: https://github.com/NOAA-EMC/NCEPLIBS-grib_util/pull/203
> 
> We can coordinate releases. We have just updated a lot of build stuff having to do with shared vs. static, and we can do a bunch of releases to get the new build stuff out everywhere.

**Pull request merged**